dask_cudf/io/orc.py

```diff
@@ -0,0 +1,26 @@
+from glob import glob
+
+from dask.base import tokenize
+from dask.compatibility import apply
+import dask.dataframe as dd
+
+import cudf
+
+
+def read_orc(path, **kwargs):
+    filenames = sorted(glob(str(path)))
+    name = "read-orc-" + tokenize(path, **kwargs)
+
+    meta = cudf.read_orc(filenames[0], **kwargs)
+
+    graph = {
+        (name, i): (apply, cudf.read_orc, [fn], kwargs)
```

Comment:
Do ORC files ever have multiple partitions in one file?  If so then this approach may not handle that well.  (cc @randerzander)

Reply:
"this approach may not handle that well" -- can you elaborate?

I've usually seen different partitions in different files.

Reply:
I believe the current ORC reader in CUDF does not expose any behavior related to partitioning:

https://github.com/rapidsai/cudf/blob/f1f667adbb54d1c7c72c5300d18f5b90b042f663/python/cudf/io/orc.py#L17-L22

Reply:
I suppose dask-cudf could potentially do this if ORC partitions can be read fully and independently.  Perhaps that is what @mrocklin is asking

Reply:
> I've usually seen different partitions in different files.

If you're in the one partition per file case then we're fine.  If you have large files with many partitions within them, then theres a lot more work to do.  

Switching to Parquet terminology (I don't know ORC well), there can be many row groups in a file, but this isn't always done. A lot of datasets use one row-group per file.  If you're in this situation then things are fine.

Reply:
Sounds like improvements for 0.8+